UPGRADE.md

```diff
@@ -28,6 +28,10 @@ Removed unused arguments:
 
 The script provided by Sulu for the piwik implementation has been updated to use mataomo path so the script is now pointing to matomo.php instead of the piwik.php file.
 
+### Removing deprecated guzzle integration
+
+As part of the update of flysystem the package `guzzlehttp/guzzle` and `guzzlehttp/promise` have been removed. If you need it you need to manually require it.
```

Comment:
The `GoogleGeolocator` and `NominatimGeolocator` not longer supporting the Guzzle Http Client instead a Symfony HttpClient instance need to give to its constructors now.